content/roadmap.md

```diff
@@ -74,17 +87,10 @@ that comes in handy in various scenarios:
   * Black/whitelisting services for adaptive sampling,
   * etc.
 
-## Tail-based Sampling
-
-Jaeger clients implement so-called _head-based sampling_, where a sampling decision is made at the root of the call tree and propagated down the tree along with the trace context. This is done to guarantee consistent sampling of all spans of a given trace (or none of them), because we don't want to make the coin flip at every node and end up with partial/broken traces. However, if 99% of all requests in the system are normal, then 99% of all traces we collect are not very interesting, and the probability of capturing really unusual traces is quite low, because at the start of the trace the platform has very little information for making a sampling decision.
-
-The alternative way to implement sampling is with _tail-based sampling_, a technique employed by some of the commercial vendors today, such as Lightstep, DataDog. With tail-based sampling, 100% of spans are captured from the application, but only stored in memory in a collection tier, until the full trace is gathered and a sampling decision is made. The decision making code has a lot more information now, including errors, unusual latencies, etc. If we decide to sample the trace, only then it goes to disk storage, otherwise we evict it from memory, so that we only need to keep spans in memory for a few seconds on average. Tail-based sampling imposes heavier performance penalty on the traced applications because 100% of traffic needs to be profiled by tracing instrumentation.
-
-You can read more about head-based and tail-based sampling either in Chapter 3 of Yuri Shkuro's book [Mastering Distributed Tracing](https://www.shkuro.com/books/2019-mastering-distributed-tracing/) or in the awesome paper ["So, you want to trace your distributed system? Key design insights from years of practical experience"](http://www.pdl.cmu.edu/PDL-FTP/SelfStar/CMU-PDL-14-102.pdf) by Raja R. Sambasivan, Rodrigo Fonseca, Ilari Shafer, Gregory R. Ganger.
-
-See issue tracker for more info: [jaeger/issues/425](https://github.com/jaegertracing/jaeger/issues/425).
-
-## Long Term Roadmap
+## Ideation
 
 * Multi-Tenancy ([mailgroup thread](https://groups.google.com/forum/#!topic/jaeger-tracing/PcxftflO4_o))
 * Cloud and Multi-DC strategy
+* Anomaly detection of flagging of traces
+* Baselining of trace metrics
```

Comment:
elaborate?

Reply:
Haven't discussed this one yet with you, but the idea would be to provide a proactive way to identify problems happening outside of a normal pattern. Building some basic baseline of the metrics and showing deviations. This is something Prometheus doesn't do today. It could belong as a Prometheus feature, but it depends how it's used inside Jaeger.